src/sentry/api/endpoints/organization_events_meta.py

```diff
@@ -30,3 +36,56 @@ def get(self, request, organization):
             raise ParseError(detail=six.text_type(error))
 
         return Response({"count": result["data"][0]["count"]})
+
+
+class OrganizationEventsRelatedIssuesEndpoint(OrganizationEventsEndpointBase, EnvironmentMixin):
+    def get(self, request, organization):
+        try:
+            params = self.get_filter_params(request, organization)
+        except OrganizationEventsError as e:
+            return Response({"detail": six.text_type(e)}, status=400)
+        except NoProjects:
+            return Response([])
+
+        possible_keys = ["transaction"]
+        lookup_keys = {key: request.query_params.get(key) for key in possible_keys}
+
+        if not any(lookup_keys.values()):
+            return Response(
+                {
+                    "detail": "Must provide one of {} in order to find related events".format(
+                        possible_keys
+                    )
+                },
+                status=400,
+            )
+
+        try:
+            projects = self.get_projects(request, organization)
+            query_kwargs = build_query_params_from_request(
+                request, organization, projects, params.get("environment")
+            )
+            query_kwargs["limit"] = 5
+            try:
+                parsed_terms = parse_search_query(
+                    'transaction:"{}"'.format(lookup_keys["transaction"])
```

Comment:
We might need to escape quotes in case some joker includes " in their transaction names.